src/main/java/com/microsoft/azure/spring/data/documentdb/repository/support/DocumentDbEntityInformation.java

```diff
@@ -72,6 +84,15 @@ public String getPartitionKeyFieldValue(T entity) {
         return partitionKeyField == null ? null : (String) ReflectionUtils.getField(partitionKeyField, entity);
     }
 
+    public IndexingPolicy createIndexingPolicy() {
+        final IndexingPolicy policy = new IndexingPolicy();
+
+        policy.setAutomatic(this.getIndexingPolicyAutomatic());
+        policy.setIndexingMode(this.getIndexingPolicyMode());
+
+        return policy;
+    }
```

Comment:
Wonder can we put this IndexingPolicy as a field of the EntifyInformation, and get the indexingPolicyAutomatic
indexingPolicyMode from the IndexingPolicy field. Instead of expose the createIndexingPolicy method, the IndexPolicy can be initialized the first time the EntifyInformation is initialized, and a get method to return the IndexingPolicy is enough?

Reply:
Copy that, will refine it.